lib/modules/plugin_cmd/index.js

```diff
@@ -5,7 +5,7 @@ class PluginCommand {
   constructor(embark, config) {
     this.embark = embark;
     this.config = config;
-    this.embarkConfig = fs.readJSONSync(this.config.embarkConfigFile);
+    this.embarkConfig = this.config.embarkConfig;
```

Comment:
My bad, I failed to see that you passed it from the options. You don't need it. You can get the config from `embark.config.embarkConfig`

Reply:
Fixed in the latest commit :)